src/pages/Authentication/Login/login.jsx

```diff
@@ -0,0 +1,136 @@
+import axios from "axios";
+import { useState, useReducer } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import { v4 as uuid } from "uuid";
+import {
+  Navbar,
+  FormContainer,
+  FormInput,
+  FormButton,
+  PasswordInput,
+} from "../../../components";
+import { loginFormValidation } from "./loginFormValidation";
+import { useDocumentTitle } from "../../../custom-hooks";
+import { errorReducer, errorActions } from "../../../reducers/errorReducer";
+import { focusHandler } from "../authUtils";
+import { useAuth } from "../../../contexts";
+import { authActions } from "../../../reducers/authReducer";
+
+const loginFormInitialErrorState = {
+  emailError: null,
+  passwordError: null,
+};
+
+const Login = () => {
+  useDocumentTitle("Login | Lens-Store");
+
+  const [loginForm, setLoginForm] = useState({ email: "", password: "" });
+
+  const [loginFormErrorState, loginFormErrorDispatch] = useReducer(
+    errorReducer,
+    loginFormInitialErrorState
+  );
+
+  const { authDispatch } = useAuth();
+
+  const [apiError, setApiError] = useState(null);
+
+  const navigate = useNavigate();
+
+  const loginHandler = async (event) => {
+    event.preventDefault();
+
+    if (loginFormValidation(loginForm, loginFormErrorDispatch)) {
+    }
+  };
+
+  const loginInputHandler = (event) => {
+    setLoginForm((prev) => ({
+      ...prev,
+      [event.target.name]: event.target.value,
+    }));
+  };
+
+  const guestLoginHandler = () => {
+    setLoginForm(() => ({
+      email: "test@gmail.com",
+      password: "test@123",
+    }));
+  };
+
+  const loginFocusHandler = (event) => {
+    focusHandler(event, loginFormErrorDispatch);
+  };
+
+  return (
+    <>
+      <Navbar />
+      <section
+        className="d-flex xy-center"
+        style={{ minHeight: "calc(100vh - 4rem)" }}
```

Comment:
you can create an utility className like `body-min-height`  for this, avoid inline style !